docs/tools/dotnet-embedding/limitations.md

```diff
@@ -44,4 +44,10 @@ Since we cannot generate accurate nullability annotations in the header files an
 
 Currently .NET Embedding does not support bitcode on iOS, which is enabled for some Xcode project templates. This will have to be disabled to successfully link generated frameworks.
 
+There are a couple reasons why .NET Embedding for iOS is not supported:
+
+* Xamarin can generate bitcode for iOS, but it is inline assembly, not "real" bitcode. Xcode can recompile iOS bitcode, but not optimize or modify it.
```

Comment:
>  Xamarin can generate bitcode for iOS

No, we could but we don't - for the reason I mentioned on slack.

Reply:
Saying "can" implies the tool can do it - which is what the customer are asking, without understanding that
*  they would not get he theoretical benefits (server side optimization);
* they would suffer longer build times (to generated bitcode) and binary sizes (which now include bitcode, before Apple re-process them)